### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by tttzach**
Refactor code and format according to previous suggestions.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 1]
```diff
@@ -56,9 +57,8 @@ private int getMax(HttpServletRequest request) {
 
   private List<String> getCommentsToDisplay(PreparedQuery results, int max) {
     List<String> comments = new ArrayList<>();
-    Iterator<Entity> iterator = results.asIterator();
-    for (int i = 0; (i < max) && (iterator.hasNext()); ++i) {
-      Entity entity = iterator.next();
+    List<Entity> entities = results.asList(FetchOptions.Builder.withLimit(max));
```
Comment (gbuenoandrade) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:60]: Neat!

Comment (tttzach) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:60]: Thanks!

Comment (tttzach) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:60]: > Can you please take a look at the [Codacy issues](https://app.codacy.com/manual/tttzach/step/pullRequest?prid=5706404)?

Thanks for pointing it out!

Comment (tttzach) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:59]: `RESOLVED`

**Pull request merged**